python/tk_multi_publish2/publish_tree_widget/custom_widget_item.py

```diff
@@ -74,6 +74,9 @@ def _on_checkbox_click(self, state):
         """
         Callback that fires when the user clicks the checkbox
         """
+        # Convert integer state to CheckState enum if necessary
+        if isinstance(state, int):
```

Comment:
Thanks @carlos-villavicencio-adsk, since the `stateChanged` signal that triggers this slot is a built-in Qt signal, we wouldn't normally expect other types, however I like the idea of adding an extra layer of defense against potential future changes in the signal.